index.html

```diff
@@ -0,0 +1,198 @@
+                     <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.
+                        Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus
+                        mus. Donec quam </p>
+                  </div>
+               </div>
+               <div class="two-blocks flex jcsb">
+                  <div class="block">
+                     <div class="items">
+                        <ul class="links">
+                           <li>
+                              <h2>Digital Strategy</h2>
+                           </li>
+                           <li>
+                              <h2>Custom Software Development</h2>
+                           </li>
+                           <li>
+                              <h2>UX Strategy & UI Design</h2>
+                           </li>
+                           <li>
+                              <h2>Cloud Migration</h2>
+                           </li>
+                           <li>
+                              <h2>IoT Application Development</h2>
+                           </li>
+                           <li>
+                              <h2>Business Data & Analytics</h2>
+                           </li>
+                           <li>
+                              <h2>Mobile Application Development</h2>
+                           </li>
+                           <li>
+                              <h2>Web Application & Software Testing</h2>
+                           </li>
+                           <li>
+                              <h2>DevOps Coaching for a Smooth Digital Transformation</h2>
+                           </li>
+                        </ul>
+                     </div>
+                  </div>
+                  <div class="block">
+                     <ul class="slides">
+                        <li>
+                           <div class="image">
+                              <img src="./img/image.png"></img>
+                           </div>
+                           <div class="text">
+                              <p>We start with asking the right questions, always with the end user top of mind. We
+                                 collaborate with you to understand your goals, help you ideate, and find ways to
+                                 innovate. Then, we develop and deliver. Our goal: provide quality work from start to
+                                 finish — and beyond.</p>
+                           </div>
+                           <div class="more">
+                              <button class="btn">Learn More</button>
+                           </div>
+                        </li>
+                        <li>
+                           <div class="image">
+                              <img src="./img/image.png"></img>
+                           </div>
+                           <div class="text">
```

Comment:
@vadym9  avoid usage of such specific naming. What if somebody will place an image here? Dont reduce amount of scenarios of usage for this widget and keep it flexible. Use `description` class, or `content`. Or any less specific. 

Do not update this PR now. Lets' merge it in current state but just keep it in mind and pay attention in your further development 👍🏼